driverstation.py

```diff
@@ -59,42 +71,56 @@ def valueChanged(table, key, value, isNew):
 GUI.setup() 
 
 
-pygame.joystick.init()
-# Assume only 1 joystick for now
-joystick = pygame.joystick.Joystick(0)
-joystick.init()
+updateFromRobot = False
+
+hasCommunication = False
+hasCode = True #TODO: make some way to check for this
```

Comment:
This will likely need to be a networktables state that we set in run.py after we successfully import robot.py?